src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -127,6 +132,24 @@ def test_azconfig_mgmt(self, resource_group, location):
 
         self.cmd('appconfig delete -n {config_store_name} -g {rg} -y')
 
+        self.cmd('appconfig show-deleted -n {config_store_name}',
+            checks=[self.check('name', '{config_store_name}'),
+                    self.check('location', '{rg_loc}'),
+                    self.check('purgeProtectionEnabled', '{enable_purge_protection}')])
+
+        self.cmd('appconfig recover -n {config_store_name} -y')
+
+        self.cmd('appconfig show -n {config_store_name}',
+            checks=[self.check('name', '{config_store_name}'),
+                    self.check('location', '{rg_loc}')])
+
+        self.cmd('appconfig delete -n {config_store_name} -g {rg} -y')
+
+        self.cmd('appconfig purge -n {config_store_name} -y')
```

Comment:
Can you add a test for the list-deleted command too?

Reply:
Added.